Tasks/VsTest/distributedtest.ts

```diff
@@ -143,11 +143,36 @@ export class DistributedTest {
             throw new Error('Failed to start Modules/DTAExecutionHost.exe.');
         });
 
+        var consolidatedCiData = {
+                    agentFailure: false,
+                    batchingType: models.BatchingType[this.dtaTestConfig.batchingType],
+                    batchsize: this.dtaTestConfig.numberOfTestCasesPerSlice,
+                    codeCoverageEnabled: this.dtaTestConfig.codeCoverageEnabled,
+                    customSlicingEnabled: utils.Helper.isNullOrUndefined(this.dtaTestConfig.customSlicingenabled) ? 'false' : 'true',
+                    environmenturi: this.dtaTestConfig.dtaEnvironment.environmentUri, 
+                    numberOfAgentsInPhase: this.dtaTestConfig.numberOfAgentsInPhase,
+                    numberOfTestCasesPerSlice: this.dtaTestConfig.numberOfTestCasesPerSlice,
+                    overridesTestRunParameters: utils.Helper.isNullOrUndefined(this.dtaTestConfig.overrideTestrunParameters) ? 'false' : 'true',
+                    pathToCustomTestAdaptersSet: utils.Helper.isNullOrUndefined(this.dtaTestConfig.pathtoCustomTestAdapters) ? 'false' : 'true',
+                    pipeline: tl.getVariable('release.releaseUri') != null ? "release" : "build",
+                    runInIsolation: this.dtaTestConfig.runTestsInIsolation,
+                    task: "VsTest",
+                    testSelection: this.dtaTestConfig.testSelection,
+                    usesMultiCoreParallel: this.dtaTestConfig.runInParallel,
+                    usesTia: this.dtaTestConfig.tiaConfig.tiaEnabled,
+                    useVsTestConsole: this.dtaTestConfig.useVsTestConsole,
+                    videoCoverageEnabled: utils.Helper.isNullOrUndefined(this.dtaTestConfig.videoCoverageEnabled) ? 'false' : 'true',
```

Comment:
why do we have videoCoverage field? i dont think we are setting this. if not, remove this

Reply:
prachi wanted this but if you're sure we are never setting this and never will i'll remove it